src/components/LoginForm/LoginForm.styled.ts

```diff
@@ -0,0 +1,13 @@
+import { Button, styled } from '@mui/material';
+import { Field } from 'formik';
+
+export const InputStyled = styled(Field)(() => {
+  return { width: '100%' };
+});
+
+export const ButtonStyled = styled(Button)(() => {
```

Comment:
Maybe `LoginFormInputStyled` and `LoginFormButtonStyled`.

If we have more of these kinds of components, it will be difficult to know which one you want to import

Reply:
changed in the latest commit.